src/CMakeLists.txt

```diff
@@ -465,7 +465,14 @@ ELSE(Q_WS_WIN)
             ${PROJECT_SOURCE_DIR}/tools/${LINUX_DIR}/ssh-*
             ${CMAKE_CURRENT_BINARY_DIR}/ssh
 		COMMAND chmod u+x ${CMAKE_CURRENT_BINARY_DIR}/ssh
-		COMMAND ${CMAKE_COMMAND} -E copy_if_different
+                COMMAND ${CMAKE_COMMAND} -E copy_if_different
+            ${PROJECT_SOURCE_DIR}/tools/${LINUX_DIR}/libcrypto.so.1.0.1e
+            ${CMAKE_CURRENT_BINARY_DIR}/libcrypto.so.1.0.1e
+                COMMAND chmod u+x ${CMAKE_CURRENT_BINARY_DIR}/libcrypto.so.1.0.1e
+                COMMAND ${CMAKE_COMMAND} -E create_symlink
+            ${CMAKE_CURRENT_BINARY_DIR}/libcrypto.so.1.0.1e
```

Comment:
... and you should register the DLL and the Symlink with `INSTALL`, otherwise it won't end up in the installer